### ISSUE TITLE
dataTable empty tr -> all following rows missing

### ISSUE THREAD

**Issue by mrNuTz** [html-to-text/node-html-to-text#282]
**Minimal HTML example**

```html
<table><tr></tr><tr><td>foo</td></tr></table>
```

**Options**

```javascript
{
  selectors: [
    {selector: 'table', format: 'dataTable'},
  ]
]
```

**Observed output**

```
```

**Expected output**

```
foo
```

**Version information**

- html-to-text: 9.0.3
- node: v18.13.0

----

**second minimal html example**

Different numbers of td's per tr also cause missing output.

```html
<table><tr><td>foo</td></tr><tr><td>foo</td><td>bar</td></tr></table>
```

**Observed output**

```
foo
foo
```

**Expected output**

```
foo
foo   bar
```

**Comment by KillyMXI** [comment]
Good catch, thank you for the report.

**Comment by KillyMXI** [comment]
Fixed in version 9.0.4
